portfolio/src/main/webapp/style.css

```diff
@@ -8,27 +8,38 @@ body {
   width: 650px;
 }
 
-intro-title {
-  font-family: cursive;
-}
-
 greeting-button {
-  background-color: rgb(151, 91, 201);
+  background-color: rgb(173, 122, 214);
   border: none;
+  border-radius: 50px;
   color: white;
-  font-size: 20px;
-  margin: 4px 2px;
+  cursor: pointer;
+  font-family: cursive;
+  font-size: 15px;
+  margin-bottom: 15px;
+  margin-top: 20px;
   padding: 15px;
   text-align: center;
 }
 
-cursive {
+#greeting-container {
+  color: black;
+  font-size: 25px;
+  margin-bottom: 50px;
+  margin-top: 40px;
+}
+
+hr {
+  border-top: 4px dotted black;
+}
+
+h1 {
   font-family: cursive;
+  margin-bottom: 50px;
 }
 
-#greeting-container {
-  font-size: 25px;
-  margin-top: 30px;
+h2, h3 {
```

Comment:
When you have multiple selectors like this, Google style guide wants each on its own line. So `h2,` on one line, and `h3 {` on the next line.